source/elements/oneDPL/source/parallel_api/execution_policies.rst

```diff
@@ -180,5 +178,55 @@ as the template argument, otherwise unspecified.
 Return a policy object constructed from ``policy``, with a new kernel name provided as the template
 argument. If no policy object is provided, the new policy is constructed from ``dpcpp_default``.
 
+.. _exec-policy-type-trait:
+
+Execution Policy Type Trait
++++++++++++++++++++++++++++
+
+oneDPL provides type trait utilities to detect at compile time its execution policy types for the purpose of
```

Comment:
```suggestion
oneDPL provides type trait utilities to detect its execution policy types at compile-time for the purpose of
```
Just a stylistic change. I feel it sounds more natural English-wise. Native speakers could help here :) And also not sure whether "compile-time" should be spelled via "-". That's how I usually spell it and how cppreference spells it

Reply:
Thanks, I will apply. The hyphen is not needed here, but would be needed if it was a compound adjective, like in "compile-time dispatch".